sklearn/covariance/_graph_lasso.py

```diff
@@ -490,14 +529,17 @@ def fit(self, X, y=None):
         else:
             self.location_ = X.mean(0)
         emp_cov = empirical_covariance(X, assume_centered=self.assume_centered)
-        self.covariance_, self.precision_, self.n_iter_ = graphical_lasso(
+        self.covariance_, self.precision_, self.costs, self.n_iter_ = _graphical_lasso(
```

Comment:
In `GraphicalLasso`, we need to modify the `Interval` for `alpha` where we should change `closed="right"` to `closed="both"` to accept `0` since the algorithm itself has a `if alpha == 0`.

I did not check yet but I think that we should have a test for `graphical_lasso` for `alpha == 0` and we should parameterize the test to also test `GraphicalLasso`.